padguidedb/padguidedb.py

```diff
@@ -111,40 +118,74 @@ async def searchdungeon(self, ctx, *, search_text):
             for page in pagify(msg):
                 await ctx.send(box(page))
 
-    @padguidedb.command()
-    @checks.is_owner()
-    async def setdungeonscriptfile(self, ctx, *, dungeon_script_file):
-        """Set the dungeon script file."""
-        self.settings.setDungeonScriptFile(dungeon_script_file)
-        await ctx.tick()
+    @padguidedb.group()
+    async def series(self, ctx):
+        """Series"""
 
-    @padguidedb.command()
-    @checks.is_owner()
-    async def setfulletlfile(self, ctx, *, full_etl_file):
-        """Set the full ETL file."""
-        self.settings.setFullETLFile(full_etl_file)
-        await ctx.tick()
+    @series.command(name="search")
+    async def series_search(self, ctx, *, search_text):
+        """Search"""
+        search_text = '%{}%'.format(search_text)
+        with self.get_cursor() as cursor:
+            sql = ('select series_id, name_en, name_ja from series'
```

Comment:
technically, anywhere you do a string.format on a raw sql string, you're vulnerable to a sql injection attack.

consider using a sql builder, e.g https://pypi.org/project/sqlbuilder/ (i just googled one randomly)